index.js

```diff
@@ -250,3 +266,15 @@ function initSession (opts, fn) {
     fn(uid, new Session(opts));
   });
 }
+
+app.on('open-file', (event, path) => {
+  let lastWindow = app.getLastFocusedWindow();
```

Comment:
Fixed the const.
I added getLastFocusedWindow to app in order to expose it to plugins developers: it could be useful also to them i guess :)